src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -78,6 +80,79 @@ describe('components/Popups/ReplyField', () => {
         });
     });
 
+    describe('getCollaborators()', () => {
+        test('should return empty list if no activeMention', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+
+            expect(instance.getCollaborators(null)).toHaveLength(0);
+        });
+
+        test('should return full collaborators list if mentionString length is less than 2', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+
+            const mockMentionShort = {
+                blockID: 'testid',
+                end: 1,
+                mentionString: '',
+                mentionTrigger: '@',
+                start: 0,
+            };
+
+            expect(instance.getCollaborators(mockMentionShort)).toMatchObject(defaults.collaborators);
+        });
+
+        test('should filter invalid items in collaborators', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+
+            expect(instance.getCollaborators(mockMention)).toHaveLength(0);
+        });
+
+        test('should filter items based on item name', () => {
+            const mockCollabs = [
+                { id: 'testid1', name: 'test1', item: { id: 'testid1', name: 'test1', type: 'user' } },
+                { id: 'testid2', name: 'test2', item: { id: 'testid2', name: 'test2', type: 'user' } },
+            ];
+            const mockMention2 = {
+                blockID: 'testid',
+                end: 6,
```

Comment:
There's some duplication of mock data between tests. It's also not clear why certain props are meaningful or why they change between tests. Is `6` here tied to something?

Reply:
> There's some duplication of mock data between tests. It's also not clear why certain props are meaningful or why they change between tests. Is `6` here tied to something?

Yeah, `mentionTrigger.length + mentionString.length = 6`